info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T07:17:09.749Z"}
info: Connected to the database. {"timestamp":"2023-06-23T07:17:09.781Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T07:17:36.668Z"}
info: Connected to the database. {"timestamp":"2023-06-23T07:17:36.715Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:03:57.283Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:03:57.526Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:06:30.169Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:06:30.208Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:06:52.840Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:06:52.911Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:08:59.079Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:08:59.132Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:10:04.074Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:10:04.130Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:10:32.771Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:10:32.898Z"}
error: 
    status - 500 
    message - Error: The client is closed
    at Commander._RedisClient_sendCommand (D:\FundooApplication\FundooApp\node_modules\@redis\client\dist\lib\client\index.js:490:31)
    at Commander.commandsExecutor (D:\FundooApplication\FundooApp\node_modules\@redis\client\dist\lib\client\index.js:188:154)
    at Commander.del (D:\FundooApplication\FundooApp\node_modules\@redis\client\dist\lib\commander.js:8:29)
    at _callee$ (D:\FundooApplication\FundooApp\src\services\/notes.service.js:9:22)
    at tryCatch (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-23T15:11:22.560Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:14:24.457Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:14:24.552Z"}
error: 
    status - 500 
    message - Error: The client is closed
    at Commander._RedisClient_sendCommand (D:\FundooApplication\FundooApp\node_modules\@redis\client\dist\lib\client\index.js:490:31)
    at Commander.commandsExecutor (D:\FundooApplication\FundooApp\node_modules\@redis\client\dist\lib\client\index.js:188:154)
    at Commander.del (D:\FundooApplication\FundooApp\node_modules\@redis\client\dist\lib\commander.js:8:29)
    at _callee$ (D:\FundooApplication\FundooApp\src\services\/notes.service.js:9:22)
    at tryCatch (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/ 
    method - POST 
    IP - ::1
   {"timestamp":"2023-06-23T15:14:30.471Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:16:05.485Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:16:05.560Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:16:26.745Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:16:26.822Z"}
error: 
    status - 500 
    message - TypeError: JSON.sprinify is not a function
    at _callee2$ (D:\FundooApplication\FundooApp\src\services\/notes.service.js:19:40)
    at tryCatch (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (D:\FundooApplication\FundooApp\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/notes/ 
    method - GET 
    IP - ::1
   {"timestamp":"2023-06-23T15:17:14.739Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:18:02.991Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:18:03.066Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:21:33.763Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:21:33.815Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:22:26.800Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:22:26.902Z"}
info: Server started at http://localhost:4001/api/v1/ {"timestamp":"2023-06-23T15:23:57.919Z"}
info: Connected to the database. {"timestamp":"2023-06-23T15:23:57.976Z"}
